src/core/index.js

```diff
@@ -275,6 +275,43 @@ util.extend( corefn, {
     return this._private.container;
   },
 
+  mount: function( container ){
+    let cy = this;
+    let _p = cy._private;
+    let options = _p.options;
+
+    if( container == null ){ return; }
+
+    if( !is.htmlElement( container ) && is.htmlElement( container[0] ) ){
+      container = container[0];
+    }
+  
+    cy.stopAnimationLoop();
+    cy.destroyRenderer();
+
+    options.container = container;
```

Comment:
Not sure if ```mount()``` needs to mutate the option values to keep the options and private fields in sync.

Reply:
Everything in options should only be used by the constructor.  The only exception would be `cy.json()`, because its output is the options.

So you probably only need to modify `_p.options.renderer`

Reply:
Alright I will remove the unnecessary operations.